src/test/pipe/pipe.shadow.config.xml

```diff
@@ -0,0 +1,27 @@
+<shadow>
+  <topology><![CDATA[<graphml xmlns="http://graphml.graphdrawing.org/xmlns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns http://graphml.graphdrawing.org/xmlns/1.0/graphml.xsd">
+  <key attr.name="packetloss" attr.type="double" for="edge" id="d4" />
+  <key attr.name="latency" attr.type="double" for="edge" id="d3" />
+  <key attr.name="bandwidthup" attr.type="int" for="node" id="d2" />
+  <key attr.name="bandwidthdown" attr.type="int" for="node" id="d1" />
+  <key attr.name="countrycode" attr.type="string" for="node" id="d0" />
+  <graph edgedefault="undirected">
+    <node id="poi-1">
+      <data key="d0">US</data>
+      <data key="d1">10240</data>
+      <data key="d2">10240</data>
+    </node>
+    <edge source="poi-1" target="poi-1">
+      <data key="d3">50.0</data>
+      <data key="d4">0.0</data>
+    </edge>
+  </graph>
+</graphml>
+]]></topology>
+  <kill time="5"/>
+  <plugin id="test_pipe" path="../target/debug/test_pipe"/>
+  <node id="testnode" quantity="1">
+    <application plugin="test_pipe" starttime="1" arguments="--shadow-passing"/>
+  </node>
+</shadow>
```

Comment:
Some changes needed here:
- remove `<kill time="5"/>`` and instead do `<shadow stoptime="5">`
- "node" -> "host"
- "application" -> "process"
- I think we are storing these config files in yaml now? So after making the above changes, use the xml2yaml converter in the tools dir, and then update the CMakeLists.txt file to convert back to XML on the fly when running the test.

Reply:
Thanks, good catch. It is now in YAML.